### PR TITLE
Rephrase the getting started summary

### PR DESCRIPTION
**Pull request by leplatrem**
@ametaireau r?

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -27,7 +26,7 @@ Run locally
 ===========
 
 Kinto is based on top of the `cliquet <https://cliquet.rtfd.org>`_ project, and
-as such, please follow the installation instructions there to get started.
+as such, please refer to its documentation regarding API and endpoints.
```
Comment (Natim) [inline README.rst]: ``refer to cliquet's documentation`` even if its is right, it is easier to understand I think.

Comment (leplatrem) [inline README.rst]: you mean : 

> as such, please refer to cliquet's documentation regarding API and endpoints.

Comment (Natim) [inline README.rst]: Yes

**Comment by ametaireau** [comment]
thanks!

**Pull request merged**